.github/workflows/code_analysis.yaml

```diff
@@ -25,17 +25,13 @@ jobs:
                         name: 'Preload php-parser Order'
                         run: php preload.php
 
-                    -
-                        name: 'Validate Max File Length'
-                        run: vendor/bin/easy-ci validate-file-length rules src tests
-
```

Comment:
The easy-ci package is not developed lately and all important checks were moved to swiss-knife package.
This is old leftover that we don't need anymore